src/main/resources/view/ReminderPersonListPanel.fxml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.ListView?>
+<?import javafx.scene.layout.VBox?>
+
+<VBox xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
Noticed you are using javafx v8. Maybe it might be better to standardise it to 11 if you are not using any particular feature from 8?